events.js

```diff
@@ -0,0 +1,29 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let userInput = {
+        list: ""   
+    }
+
+    let form = document.querySelector("form")
+    form.addEventListener("change", (e) =>{
+        userInput[e.target.id] = e.target.value
```

Comment:
If you're going to use the `id` of the tag, would need to add an id yourself and make them different based on each list item